FlowCryptTests/KeySettingsItemTest.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  KeySettingsItemTest.swift
+//  FlowCryptTests
+//
+//  Created by Anton Kharchevskyi on 12/13/19.
+//  Copyright © 2019 FlowCrypt Limited. All rights reserved.
+//
+
+import XCTest
+
+class KeySettingsItemTest: XCTestCase {
+    func test_key_initializer() {
+
```

Comment:
We should either implement or remove this test

Reply:
Thanks. Added test and fixed implementation. Nice catch